config.yaml

```diff
@@ -0,0 +1,13 @@
+options:
+  ha-cluster-vip:
+    type: string
+    description: |
+      Virtual IP for the charm to use with the HA Cluster subordinate charm
+      Mutually exclusive with ha-cluster-dns.
```

Comment:
How do I specify multiple vips?

Reply:
Updated to add text about whitespace separation.